data/tutorials/getting-started/gs_02_fst_program.md

```diff
@@ -332,7 +341,7 @@ Entering directory '/home/cuihtlauac/caml/ocaml.org'
 My name is Minimo
 ```
 
-Note that `minimo.exe` is not a file name. This is how Dune is told to compile the `minimo.ml` file using OCaml's _binary_ compiler, because it also has a second one: the _byte code_ compiler.
+Note that `minimo.exe` is not a file name. This is how Dune is told to compile the `minimo.ml` file using OCaml's native compiler instead of the bytecode compiler. As a fun fact, note that an empty file is valid OCaml syntax, you can use that to reduce `minimo` even more; of course it will not display anything, but it will be a valid project.
```

Comment:
```suggestion
Note that `minimo.exe` is not a file name. This is how Dune is told to compile the `minimo.ml` file using OCaml's native compiler instead of the bytecode compiler. As a fun fact, note that an empty file is valid OCaml syntax. You can use that to reduce `minimo` even more; of course, it will not display anything, but it will be a valid project!
```